CodeReviewChallenge.java

```diff
@@ -0,0 +1,59 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Collections;
+
+public class Class1 {
+    public final String SPECIAL_CHARACTER = "a";
+
+    public List<String> method(String paragraph) {
+        List<String> shortestWords = new ArrayList<>();
+        String[] sentences =   paragraph.split(".");
+
+        for (int i=0;i<sentences.length;i++) {
+            String[] words = sentences[i].split(" ");
+            String shortestWord = words[0];
```

Comment:
Should check if the array 'words' is not empty. And also the first word might not contain an 'a'.